10_date_timestamps_and_lies.md

```diff
@@ -145,31 +143,10 @@ df.select(to_date($"lastUpdate").as("lastUpdate"))
 // +----------+-----+
 ```
 
-and
-
-```scala
-df.select(to_date($"lastUpdate").as("lastUpdate"))
-   .groupBy($"lastUpdate").count
-   .orderBy($"lastUpdate").printSchema
-```
-
-gives the following:
-
-```scala
-// root
-//  |-- lastUpdate: date (nullable = false)
-//  |-- count: long (nullable = false)
-```
-
 whereas,
 
 ```scala
-df.select(to_date($"lastUpdate").as("lastUpdate"))
-  .groupBy($"lastUpdate")
-  .count
-  .orderBy($"lastUpdate")
-  .filter($"lastUpdate".isNull)
-  .show
+df.filter($"lastUpdate".isNull).show
```

Comment:
I think that you are right. I believe that I've added the groupBy's to show that there is actually null and not null values and ordering was enable to be sure to get the values I wanted. The df is big.